src/Snowflakes/Player.php

```diff
@@ -16,72 +19,68 @@ class Player
 	 * @var float $fps
 	 * @var str $out
 	*/
-	public function __construct($width, $height, $numMolecules, $maxSteps, $fps = self::DEFAULT_FPS, $out = self::DEFAULT_VIDEO_FILE_NAME) {
-		$this->engine = new Engine($width, $height, $numMolecules);
-		$this->maxSteps = $maxSteps;
-		$this->frameNameLength = ceil(log10($this->maxSteps));
+	public function __construct($width, $height, $numMolecules, $maxFrames, $stopWhenCrystalized, $displayNthStep, $fps = self::DEFAULT_FPS, $out = self::DEFAULT_VIDEO_FILE_NAME) {
+		$this->engine = new Engine($width, $height, $numMolecules, [[80, 120], [100, 100], [30, 50], [180, 160]]);
```

Comment:
I don't think it is a good default for centers of crystallization. You don't know whether these points are inside the frame.

Reply:
right you are